site/en/blog/renderingng-data-structures/index.md

```diff
@@ -0,0 +1,689 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Key data structures and their roles in RenderingNG'
+description: >
+  This post explains the components of the RenderingNG architecture,
+  and how the rendering pipeline flows through them.
+date: 2021-08-17
+authors:
+  - chrishtr
+---
```

Comment:
https://github.com/dalecurtis (image good enough?)

https://github.com/bfgeek (image good enough?)

https://github.com/kojiishi (image good enough?)

https://github.com/progers (image good enough?)

https://github.com/una

Reply:
Still working on Daniel Cheng and Kyle Charbonneau

Reply:
Please also add the sentences attributing the authors to the various sections I specified in the google doc.

Reply:
https://github.com/zetafunction is Daniel Cheng. He's going to upload an image soon.